airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/util/PostgresSslConnectionUtils.java

```diff
@@ -5,6 +5,7 @@
 package io.airbyte.integrations.util;
```

Comment:
I think the location of the file is confusing - is this a postgres source only? is this a base class?
Most of the functionality is not used in source

Reply:
It's not great, but it can be part of a future refactor. There are a lot of constants here that are actually used in PostgresSource. (PARAM_MODE, PARAM_SSL, PARAM_SSL_MODE etc. So it's a good first place to put this

Reply:
Consts are the same, it's just the non of the functionality in this file should be used. 
I'm OK with moving this here for now to simplify but it will need probably another cleanup